tv_apps/smart-system/js/homescreen_window_manager.js

```diff
@@ -137,11 +137,12 @@
             //
             // In this case, HomescreenWindowManager will not close and reset
             // _activeHome. We should call closeHomeApp to reset the variable
-            // and focus back.
-            this.closeHomeApp();
+            // and focus back. -> switchApp in app window manager will close it.
+            // this.closeHomeApp();
 
             this._underlayApp.focus();
             this._underlayApp = null;
+            this._activeHome = null;
           } else if (this._underlayApp) {
```

Comment:
Another regression may be here. We should call `_underlayApp.close('immediate')` in search app because we don't want to have additional animation on search app.